Sources/WalletConnectSign/Engine/Common/SessionEngine.swift

```diff
@@ -59,28 +59,29 @@ final class SessionEngine {
         let chainRequest = SessionType.RequestParams.Request(method: request.method, params: request.params)
         let sessionRequestParams = SessionType.RequestParams(request: chainRequest, chainId: request.chainId)
 
-        let rpcRequest = RPCRequest(method: SignProtocolMethod.sessionRequest.method, params: sessionRequestParams)
-        try await networkingInteractor.request(rpcRequest, topic: request.topic, tag: SignProtocolMethod.sessionRequest.requestTag)
+        let rpcRequest = RPCRequest(method: SessionRequestProtocolMethod().method, params: sessionRequestParams)
```

Comment:
SessionRequestProtocolMethod() to variable? To avoid typo later

Reply:
but may be it better to isolate RPCRequest creation inside Networking package. But it another task of course